README.md

```diff
@@ -33,7 +33,6 @@ git clone \
 - If using [vim-plug][3]: `Plug 'mfussenegger/nvim-lint'`
 - If using [packer.nvim][4]: `use 'mfussenegger/nvim-lint'`
 
-
```

Comment:
Could you please revert the unrelated formatting changes?

Reply:
sure, I reverted them